lib/butter.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
+var BUTTER_CLIENT_VERSION = '1.1.3';
+
 var axios = require('axios');
-var config = require('../package.json');
```

Comment:
@beautifulcoder Any idea on why that relative path doesn't work?

Reply:
Hard to say, my gut feeling is the client bundler doesn't include this file so it fails. Likely not ButterCMS's bundler but webpack.